README.md

```diff
@@ -208,6 +208,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 
 * [Gson](https://code.google.com/p/google-gson/) - Serializes Java objects to JSON and vice versa. Good performance with on-the-fly usage.
 * [Jackson](http://wiki.fasterxml.com/JacksonHome) - Similar to GSON but has performance gains if you need to instantiate the library more often.
+* [Fastjson](https://github.com/alibaba/fastjson) - Fastjson is a JSON processor (JSON parser + JSON generator) written in Java
```

Comment:
The library is called `fastjson`. The explanation of an JSON processor isn't necessary since it is explained in the section. Also you are missing a `.` at the end of the description.